Common/Product/TestAdapter/TestContainer.cs

```diff
@@ -28,7 +28,7 @@ internal class TestContainer : ITestContainer {
 
         public TestContainer(ITestContainerDiscoverer discoverer, string source, string projectHome, string projectName, int version, Architecture architecture, bool isWorkspace) {
             Discoverer = discoverer;
-            Source = source.ToLower(); // Make sure source matches pytest discovery test file paths.
+            Source = source; // Make sure source matches discovery new TestCase source.
             Version = version;
```

Comment:
Previously ToLower() was added to fix a previous bug of the pytest test adapter returning lowercase sourcePath's and Test Explorer would then not clear tests when projects were unloaded.

This ToLower() is breaking unittest execution.. so now reverting and instead changing pytest discover to lookup the matching TestContainer.Source instead of using the sourcePath returned from discovery